### DISCUSSION TITLE
Joining the team

### DISCUSSION THREAD

**Discussion by zaid2sultan** [ROM-builders/temporary#24404] [General]
Hi Sir Apon77,

If you may, i would like to get into the team please.

My [Device Tree](https://github.com/zaid2sultan/android_device_xiaomi_lavender-p/tree/pixelos13-lav) (as a proof to #learn)
My [Manifest ](https://github.com/zaid2sultan/local_manifests/blob/pixelos13-lav/local_manifest.xml) 

Telegram: Zaid666
Github: zaid2sultan

**Comment by hipexscape** [comment]
> Hi Sir Apon77,
> 
> If you may, i would like to get into the team please.
> 
> My [Device Tree](https://github.com/zaid2sultan/android_device_xiaomi_lavender-p/tree/pixelos13-lav) (as a proof to #learn)
> My [Manifest ](https://github.com/zaid2sultan/local_manifests/blob/pixelos13-lav/local_manifest.xml) 
> 
> Telegram: Zaid666
> Github: zaid2sultan

![GitHub.png](https://github.com/ROM-builders/temporary/assets/122148266/7898e5a8-a20c-47d3-b0f0-8ff1013d3166)

This you consider as a tree ?

**Reply by zaid2sultan** [reply]
Why

**Reply by hipexscape** [reply]
> Why

Literally you have done 2 commits with no good commit label and no authorship

Follow #learn in Rom-Builders Group

**Comment by Shakib-BD** [comment]
[#learn](t.me/MissRose_bot?start=notes_-1001412119018_1184686)

please learn git command line, and respect commit history, also learn cherry pick, commiting new features,
